spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryUtil.java

```diff
@@ -48,4 +53,61 @@ private static Properties loadBuildProperties() {
   public static List<String> optimizeLoadUriListForSpark(List<String> uris) {
     return BigQueryUtil.optimizeLoadUriList(uris, ".*/part-", "-[-\\w\\.]+");
   }
+
+  /**
+   * Checks whether temporaryGcsBucket or persistentGcsBucket parameters are present in the config
+   * and creates a org.apache.hadoop.fs.Path object backed by GCS. When the indirect write method in
+   * dsv1 is used, the data is written first to this GCS path and is then loaded into BigQuery
+   *
+   * @param config SparkBigQueryConfig
+   * @param conf Hadoop configuration parameters
+   * @param applicationId A unique identifier for the Spark application
+   * @return org.apache.hadoop.fs.Path object backed by GCS
+   * @throws IOException
+   */
+  public static Path createGcsPath(
+      SparkBigQueryConfig config, Configuration conf, String applicationId) throws IOException {
+    Path gcsPath;
+    Preconditions.checkArgument(
+        config.getTemporaryGcsBucket().isPresent() || config.getPersistentGcsBucket().isPresent(),
+        "Temporary or persistent GCS bucket must be informed.");
```

Comment:
Please change the message to "Either Temporary or persistent GCS bucket must be set"

Reply:
Done